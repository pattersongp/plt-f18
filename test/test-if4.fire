func bool foo = (int x) => {

if (x > 10)
{
return true;
}
elif
{
return true;
}
else
{
return false;
}

}