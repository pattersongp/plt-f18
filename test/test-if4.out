100
checkpoint 1
500
